junit/src/main/java/org/kaazing/k3po/junit/rules/K3poRule.java

```diff
@@ -65,12 +67,14 @@
     private String scriptRoot;
     private URL controlURL;
     private SpecificationStatement statement;
+    private List<String> classOverriddenProperties;
 
     /**
      * Allocates a new K3poRule.
      */
     public K3poRule() {
         latch = new Latch();
+        classOverriddenProperties = new ArrayList<>();
```

Comment:
Why do we need this? Nothing ever seems to get set into it, and the annotation definition appears to imply it can only be used on a method. Having said that, maybe it would be useful to allow it at class level too. Was that the intention?